examples/demo_formats.py

```
@@ -5,7 +5,7 @@
 
 from pathlib import Path
 
-from markdown_lab.markdown_lab_rs import OutputFormat, convert_html
+from markdown_lab.markdown_lab_rs import convert_html_to_format
```

Pylint (3.3.7)

[error] Unable to import 'markdown_lab.markdown_lab_rs'

(E0401)